### PR TITLE
[chip testplan] Updates based on initial review

### PR DESCRIPTION
**Pull request by sriyerg**
Review notes are captured here:
https://docs.google.com/document/d/1OhPP-HjciwKpIh0wWt1xqPqPf0Y0powmmww6xekwMeE/

This first round of reviews focused on UART, GPIO, HMAC and the PLIC
blocks.

This PR contains updates based on review feedback in addition to some
minor fixes.

Signed-off-by: Srikrishna Iyer <sriyer@google.com>

### CODE REVIEW

File: hw/top_earlgrey/data/chip_testplan.hjson [hunk 1]
```diff
@@ -713,7 +741,14 @@
       name: chip_aes_idle
       desc: '''Verify AES idle signaling to clkmgr.
 
-            Details TBD.
+            - Write the AES clk hint to 1 within clkmgr to indicate AES clk is ready to be gated.
+              Verfy that the AES clk hint status within clkmgr reads 0 (AES is idle).
```
Comment (cindychip) [inline hw/top_earlgrey/data/chip_testplan.hjson:745]: Verfy -> Verify, and in line 747 and 750

File: hw/top_earlgrey/data/chip_testplan.hjson [hunk 2]
```diff
@@ -722,28 +757,27 @@
...
       tests: []
     }
     {
       name: chip_hmac_idle
-      desc: '''Verify HMAC idle signaling to clkmgr.
+      desc: '''Verify the HMAC clk idle signal to clkmgr.
 
-            Details TBD.
+            - Write the HMAC clk hint to 1 within clkmgr to indicate HMAC clk is ready to be gated.
+              Verfy that the HMAC clk hint status within clkmgr reads 0 (HMAC is idle).
```
Comment (cindychip) [inline hw/top_earlgrey/data/chip_testplan.hjson:774]: Verfy -> Verify and below

**Review by cindychip — APPROVED** [review]
LGTM with a small typo

File: hw/top_earlgrey/data/chip_testplan.hjson (hunk 1 above)
Comment (sriyerg) [reply hw/top_earlgrey/data/chip_testplan.hjson]: Done, thanks!

**Pull request merged**